build_image.sh

```diff
@@ -27,6 +24,11 @@ if [ "$IMAGE_TYPE" = "onie" ]; then
             if [ -f ./device/$VENDOR/$PLATFORM/installer.conf ]; then
                 cp ./device/$VENDOR/$PLATFORM/installer.conf ./installer/x86_64/platforms/$PLATFORM
             fi
+
+            if [ "$IMAGE_TYPE" = "raw" ] && [ -f ./device/$VENDOR/$PLATFORM/nos_to_sonic_grub.cfg ]; then
+                sed -i -e "s/%%IMAGE_VERSION%%/$IMAGE_VERSION/g" ./device/$VENDOR/$PLATFORM/nos_to_sonic_grub.cfg
```

Comment:
the IMAGE_VERSION is not replaced in the nos_to_sonic_grub.cfg, can you check?

Reply:
In my local build, it is getting reset after make target/sonic-broadcom.raw:

padman@ubuntu-16:/gvob/sdchk$ diff ./sonic-buildimage/device/dell/x86_64-dell_s6100_c2538-r0/nos_to_sonic_grub.cfg.base ./sonic-buildimage/device/dell/x86_64-dell_s6100_c2538-r0/nos_to_sonic_grub.cfg
22,23c22,23
<         linux   /image-%%IMAGE_VERSION%%/boot/vmlinuz-3.16.0-4-amd64 root=/dev/sda8 rw console=tty0 console=ttyS1,9600n8 loop=image-%%IMAGE_VERSION%%/fs.squashfs loopfstype=squashfs apparmor=1 security=apparmor
<         initrd  /image-%%IMAGE_VERSION%%/boot/initrd.img-3.16.0-4-amd64
---
>         linux   /image-master.0-2d3b064/boot/vmlinuz-3.16.0-4-amd64 root=/dev/sda8 rw console=tty0 console=ttyS1,9600n8 loop=image-master.0-2d3b064/fs.squashfs loopfstype=squashfs apparmor=1 security=apparmor
>         initrd  /image-master.0-2d3b064/boot/initrd.img-3.16.0-4-amd64
31,32c31,32
<         linux   /image-%%IMAGE_VERSION%%/boot/vmlinuz-3.16.0-4-amd64 root=/dev/sda8 rw console=tty0 console=ttyS1,9600n8 loop=image-%%IMAGE_VERSION%%/fs.squashfs loopfstype=squashfs apparmor=1 security=apparmor
<         initrd  /image-%%IMAGE_VERSION%%/boot/initrd.img-3.16.0-4-amd64
---
>         linux   /image-master.0-2d3b064/boot/vmlinuz-3.16.0-4-amd64 root=/dev/sda8 rw console=tty0 console=ttyS1,9600n8 loop=image-master.0-2d3b064/fs.squashfs loopfstype=squashfs apparmor=1 security=apparmor
>         initrd  /image-master.0-2d3b064/boot/initrd.img-3.16.0-4-amd64
padman@ubuntu-16:/gvob/sdchk$